backend/.gitignore

```diff
@@ -33,3 +33,6 @@ coverage
 /vendor/*
 .bundle/config
 dockerfile_rails.tar
+
+.env.local
+.env.test
```

Comment:
Add an empty line at the bottom. 
There are extensions that do it every time you save a file.